scripts/migrate.js

```diff
@@ -1,16 +1,16 @@
 // scripts/migrate.js
-const { migrate } = require('postgres-migrations');
-const path = require('path');
+const { migrate } = require("postgres-migrations");
```

Comment:
I know the track says its play to copy this in to the codebase, but it doesn't mean you should leave in questions like the one on line 7 which are only there to help you think about possible optimisations. Think of it from the perspective of a possible new employer, things like this and the commented out title at the top of the file make it especially clear that this was pasted in from somewhere which doesn't leave a great impression

I would always say its much more beneficial to you while learning these things to get into the idea of typing your work out as it helps you get accustomed to the flow of work, its more engaging mentally, and you get the same code in the end. To be clear, this isnt a criticism since we said its okay to copy paste in this instance, just a suggestion from someone whos being doing this for years and has noticed the difference in doing that :)